rplugin/python3/denite/source/defx/bookmarks.py

```diff
@@ -0,0 +1,73 @@
+import os
+import sys
+
+sys.path.append(
+    os.path.dirname(os.path.dirname(os.path.dirname(os.path.dirname(__file__))))
+)
+
+import dirmark.util as dm
+from denite.source.base import Base
+
+BOOKMARKS_HIGHLIGHT_SYNTAX = [
+    {'name': 'bm_name', 'link': 'Keyword', 're': r'^\[.*\]\ze\s'}
```

Comment:
You should read other sources.
You must not use `^` in highlight pattern.

Reply:
Thanks!